src/Funnel.sol

```diff
@@ -35,72 +46,79 @@ contract Funnel is IFunnel, NativeMetaTransaction, MetaTxContext, Initializable
     // owner => spender => renewableAllowance
     mapping(address => mapping(address => RenewableAllowance)) rAllowance;
 
-    /*//////////////////////////////////////////////////////////////
-                            EIP-2612 STORAGE
-    //////////////////////////////////////////////////////////////*/
+    //////////////////////////////////////////////////////////////
+    ///                        EIP-2612 STORAGE
+    //////////////////////////////////////////////////////////////
 
-    uint256 internal INITIAL_CHAIN_ID;
+    // initial_chain_id to be set during initiailisation
+    uint256 internal initial_chain_id;
```

Comment:
```suggestion
    uint256 internal INITIAL_CHAIN_ID;
```
I would prefer caps since semantically it's a constant, but we can't add constant/immutable since we use Initialiser pattern.

Reply:
Understand the concern but since this is a style guide fix, I'd try to follow strictly to it. We can leave a comment though if we really don't think this should be changed to snake_case